catalog/app/components/MIcon/index.js

```diff
@@ -12,25 +15,42 @@ const Adjust = styled.span`
   top: ${(props) => props.drop || 0}
 `;
 
+const spin = keyframes`
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+`;
+
+const animation = css`
+  animation: ${spin} 1s linear infinite;
```

Comment:
if this is for animation spinners, try to use `components/Spinner` for this. if you need custom icons for anything give me the fontawesome code and i'll add it to our fortawesome distro.